app/controllers/works_controller.rb

```diff
@@ -16,11 +16,9 @@ def new
 
   def create
     @work_form = WorkForm.new(work_params)
-    # The deposit param determines whether extra validations for deposits are applied.
     if @work_form.valid?
-      # Setting the deposit_job_started_at to the current time to indicate that the deposit job has started and user
-      # should be "waiting".
-      work = Work.create!(title: @work_form.title, user: current_user, deposit_job_started_at: Time.zone.now)
+      work = @work_form.save!(user: current_user)
+      # The deposit param determines whether extra validations for deposits are applied.
```

Comment:
Saving is delegated to the form so it can deal with persisting associated models. This is done because we're deviating from the Rails way a bit by using AM forms instead of straight up AR models, which handle nested attributes more smoothly.